ind_rnn_cell.py

```diff
@@ -60,13 +60,16 @@ def build(self, inputs_shape):
       "input_%s" % rnn_cell_impl._WEIGHTS_VARIABLE_NAME,
       shape=[input_depth, self._num_units])
 
+    recurrent_init = init_ops.random_uniform_initializer(
```

Comment:
Would it make sense to make the `recurrent_init` a constructor argument like in `GRUCell` and `LSTMCell` with the default being uniform init? It seems to make a big difference, so people might want to experiment with it.

Reply:
Definitely, will look into that.